storage/filestorage/filestorage.go

```diff
@@ -61,6 +64,7 @@ func (s *FileStorage) Path(image string) (string, bool) {
 
 // Get implements imagor.Storage interface
 func (s *FileStorage) Get(_ *http.Request, image string) (*imagor.Blob, error) {
+	defer prometheus.NewTimer(storage.OperationHistorgram.WithLabelValues("file", "get")).ObserveDuration()
```

Comment:
I would have preferred to use a wrapper like `ObservableStorage()` that calls the real storage functions, but because the storage interface is in the `imagor` package (at the root of the repo) doing that causes an import cycle.